assignments/callbacks.js

```diff
@@ -26,30 +27,68 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+    // getLength passes the length of the array into the callback.
+    return cb(arr.length);
 }
 
+getLength(items, (length) => {
+    console.log(length);
+})
+
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
-}
+    // last passes the last item of the array into the callback.
+    return cb(arr[arr.length - 1]);
+};
+
+last(items, (lastElement) => {
+    console.log(lastElement);
+})
 
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+    // sumNums adds two numbers (x, y) and passes the result to the callback.
+    return cb(x + y);
 }
 
+sumNums(1, 2, (sum) => {
+    console.log(sum);
+});
+
 function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+    // multiplyNums multiplies two numbers and passes the result to the callback.
+    return cb(x * y);
 }
 
+multiplyNums(2, 2, (product) => {
+    console.log(product);
+});
+
 function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
+    // contains checks if an item is present inside of the given array/list.
+    // Pass true to the callback if it is, otherwise pass false.
+    return cb(list.includes(item));
 }
 
+contains('Pencil', items, (boolValue) => {
+    console.log(boolValue);
+})
+
+contains('Pickle', items, (boolValue) => {
+    console.log(boolValue);
+})
```

Comment:
one note just starting off, for the case of keeping it DRY, with the way you used your callbacks you could have made one callback `const logger = value => console.log(value)` and applied that everywhere

Reply:
Makes sense. More elegant!